src/LICENSE.txt

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
the only license we need should be at the root, so remove all the licenses inside the folders.

Reply:
done

Reply:
you have to push the chnages, the license at `src/LICENSE.txt` still exists in this PR, we wan't to remove it.